_artifacts/document-attributes-global.adoc

```diff
@@ -9,7 +9,7 @@ ifdef::cloud-service[]
 :url-productname-short: openshift_ai
 :vernum: 1
 :install-package: Add-on
-:openshift-platform: OpenShift Dedicated
+:openshift-platform: OpenShift
 :openshift-platform-url: openshift-dedicated
```

Comment:
Have you checked whether we're using `openshift-platform-url` anywhere? If not, we should remove it.

Reply:
I found a handful of occurrences (all upstream). We'll have to fix those first and then remove the attribute. I'll create a separate JIRA for that work.

Reply:
I was curious what those occurrences might be, so I also took a look. As far as I can see, the upstream instances are all in self-managed conditional statements, and so would use this version of the attribute: https://github.com/opendatahub-io/opendatahub-documentation/blob/34d6107d0fbba01096edc76b55ab6693aba7768a/_artifacts/document-attributes-global.adoc#L26

Upstream, at least, I couldn't see any cloud service conditional statements that use https://github.com/opendatahub-io/opendatahub-documentation/blob/34d6107d0fbba01096edc76b55ab6693aba7768a/_artifacts/document-attributes-global.adoc#L13.

Let's also do that check downstream.